lapis2/src/main/kotlin/org/genspectrum/lapis/OpenApiDocs.kt

```diff
@@ -137,6 +148,24 @@ fun buildOpenApiSchema(sequenceFilterFields: SequenceFilterFields, databaseConfi
                             .properties(aminoAcidMutationProportionSchema()),
                     ),
                 )
+                .addSchemas(
+                    NUCLEOTIDE_INSERTIONS_RESPONSE_SCHEMA,
+                    lapisResponseSchema(
+                        Schema<String>()
+                            .type("object")
+                            .description("Nucleotide Insertion data.")
+                            .properties(nucleotideInsertionSchema()),
+                    ),
```

Comment:
I still wonder whether we could simply write something like `Schema.forImplementationClass<NucleotideInsertionResponse>`. The springdocs lib seems to be able to do it from annotations, so it should be possible somehow. Maybe someone™ should take some time to dive into the framework in a debugging session and check whether it offers a possibility...